pom.xml

```diff
@@ -68,7 +68,7 @@
 
   <properties>
     <rename.offset>org.apache.phoenix.thirdparty</rename.offset>
-    <guava.version>29.0-android</guava.version>
+    <guava.version>31.0.1-android</guava.version>
```

Comment:
Do we still want to stick to android not 31.0.1-jre because of java 7 compatibility?

Reply:
Exactly.
There is no special provision in thridparty in Java 7, apart from using guava-android.